src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -23,11 +48,43 @@ public String getCompanyNumber() {
         return companyNumber;
     }
 
+    public Map<String, String> getAttachments() {
+        return attachments;
+    }
+
+    public String getReferenceNumber() {
+        return referenceNumber;
+    }
+
+    public String getCustomerEmail() {
+        return customerEmail;
+    }
+
+    public String getReason() {
+        return reason;
+    }
+
+    private Map<String, String> buildAttachmentsMap(List<Attachment> attachments) {
+        Map<String, String> attachmentsMap = new HashMap<>();
+        for (Attachment attachment : attachments) {
+            String name = attachment.getName();
+            Links links = attachment.getLinks();
+            if(links != null) {
+                attachmentsMap.put(name, links.getLink(CoreLinkKeys.SELF));
```

Comment:
the link will need to be the download link so the users can cut and paste them into a browser for downloading (same as the ones that go in the email that have a download prefix added to them to make them usable)